src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -516,17 +516,23 @@ namespace ngraph
                     return;
                 }
                 auto reshape = static_cast<const op::Reshape*>(node);
+
+                if (out[0].get_name() == args[0].get_name() && !reshape->get_is_transpose())
```

Comment:
since we don’t have a in place transpose.  `out[0].get_name() == args[0].get_name()` should covered everything.

Reply:
If we add one we will need to remember to put this back. Does it hurt anything having it here?

Reply:
All right, we can leave it for later.